bin/wasm-node/rust/src/ffi.rs

```diff
@@ -512,29 +512,53 @@ fn init(
     ));
 }
 
+#[derive(Debug)]
+pub(crate) struct Request {
+    pub(crate) method_name: Box<[u8]>,
```

Comment:
```suggestion
    pub(crate) json_rpc_request: Box<[u8]>,
```

It's not just a method name